recipes/rjchallis-assembly-stats/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "17.02" %}
+
+package:
+  name: rjchallis-assembly-stats
+  version: {{ version }}
+
+source:
+  url: https://github.com/rjchallis/assembly-stats/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 92dd9fb958873cdae153521a29e27ad16dd85402510e8c17ccc8987a9222f37d
+
+build:
+  number: 0
+  noarch: generic
+
+license_file: LICENSE
+
+requirements:
+  run:
+    - perl
+    - perl-json
+
+test:
+  commands:
+    - perl -e 'print ">test\n"; print "A" x 1000' | asm2stats.pl
+    - perl -e 'print ">test\n"; print "A" x 1000' | asm2stats.minmaxgc.pl
+
+about:
+  home: "https://github.com/rjchallis/assembly-stats"
+  summary: Assembly metric visualisations to facilitate rapid assessment and comparison of assembly quality.
+  license: MIT License
```

Comment:
can you please use here SPDX IDs: https://spdx.org/licenses/MIT.html

Reply:
did not know what went there and grepped for examples, I guess they were misleading, thanks!